bintray.gradle

```diff
@@ -0,0 +1,26 @@
+//
+// This file is to be applied to some subproject.
+//
+
+apply plugin: 'com.jfrog.bintray'
+
+bintray {
+    user = bintray_user
+    key = bintray_api_key
+    configurations = ['archives']
+    pkg {
+        repo = 'org.jmonkeyengine'
```

Comment:
updates.jmonkeyengine.com/maven is not a bintray repo
upload directly to jcenter is not the recommended path, we upload into our own repository, and then linked artifacts are availble in every linked repository like jcenter. eg I upload my artifact in my own repository and for artifacts linked into jmonkeyengine/contrib repository or jcenter.
see [Exposing Your Package to the World through More Popular Repositories (Linking a Package)](https://bintray.com/docs/usermanual/uploads/uploads_exposingyourpackagetotheworldthroughmorepopularrepositorieslinkingapackage.html)